modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1594,13 +1594,19 @@
 			<description name="Replaces with `compileInclude` the configuration attribute for dependencies in `build.gradle` that are listed at `Include-Resource` property at `bnd.bnd` associated file." />
 			<weight>2</weight>
 		</check>
+		<check name="UpgradeJavaAddFDSTableSchemaFieldCheck">
```

Comment:
@michaelmdelcavalcanti, shouldn't the commit message mention `UpgradeSourceProcessor` instead of `JavaSourceProcessor`?

Reply:
The correctly is UpgradeSourceProcessor, I fixed it, thx.